### PR TITLE
Sync file from nfs

### PR DESCRIPTION
**Pull request by pb-cdunn**
Syncing stderr from nfs is sometimes slow.
```
[INFO] 2015-11-05 14:51:34,380Z [pbcommand.models.report write_json 687] Wrote report /pbi/dept/secondary/siv/smrtlink/smrtlink-bihourly/smrtsuite/userdata/jobs_root/000/000064/workflow/report-tasks.json
[ERROR] 2015-11-05 14:51:34,391Z [status.pbsmrtpipe.driver __exe_workflow 484] Failed with exit code 1
[ERROR] 2015-11-05 14:51:34,393Z [pbsmrtpipe.driver __exe_workflow 485] Failed with exit code 1

[WARNING] 2015-11-05 14:51:34,395Z [pbsmrtpipe.driver _get_last_lines_of_stderr 151] Unable to extract stderr from Failed with exit code 1 Error 
```
Calling os.listdir() might solve this.

(Discussed with @gconcepcion.)

### CODE REVIEW

**Comment by mpkocher** [comment]
Please use `pbsmrtpipe.utils.nfs_refresh` (or improve that implementation if necessary). 

I would like to centralized these NFS related hacks to a single function.

**Comment by pb-cdunn** [comment]
Ok.

**Comment by pb-cdunn** [comment]
I haven't tested this yet. I added an exception which I have not seen yet, and I have to use my special debug branch of pbsmrtpipe to figure out why...

**Comment by pb-cdunn** [comment]
Whoa. Bad commit. Wait...

File: pbsmrtpipe/utils.py [hunk 1]
```diff
@@ -49,31 +49,64 @@ def _validate_resource(func, resource):
...
-    if os.path.exists(path):
-        return True
+    return rcode == 0
 
+
+def nfs_refresh(path, ntimes=3):
     # re-try
-    for i in xrange(ntimes):
-        if _nfs_exists_check(path):
+    while True:
```
Comment (mpkocher) [inline pbsmrtpipe/utils.py]: Why did you change this?

**Comment by pb-cdunn** [comment]
This works for me.

File: pbsmrtpipe/utils.py (hunk 1 above)
Comment (pb-cdunn) [inline pbsmrtpipe/utils.py]: The delay was occurring after the last try, which was just wasting time.

I have seen you make the same mistake elsewhere, so I was hoping you noticed this.

Comment (mpkocher) [inline pbsmrtpipe/utils.py]: What exactly is wrong with this?
```
   for i in xrange(ntimes):
        if _nfs_exists_check(path):
            return True
        #
        time.sleep(1.0)
``` 

https://github.com/PacificBiosciences/pbsmrtpipe/blob/master/pbsmrtpipe/utils.py#L73

Comment (pb-cdunn) [inline pbsmrtpipe/utils.py]: The delay was occurring after the last try, which was just wasting time.

I have seen you make the same mistake elsewhere, so I was hoping you noticed this.

* check
* sleep
* check
* sleep
* check
* sleep
* fail

The last sleep has no purpose.

Comment (mpkocher) [inline pbsmrtpipe/utils.py]: Thanks for clarifying. 

The common use is this:

```
nfs_refresh(path)
r = ReferenceSet(path)
```
